### PR TITLE
Fix the drupal backends

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

File: anitya/lib/backends/drupal6.py [hunk 1]
```diff
@@ -59,9 +62,13 @@ def get_versions(cls, project):
...
         '''
+        name = project.name
+        if 'drupal6:' in name.lower():
+            name = name[len('drupal6:'):].strip()
+
         url_template = 'https://www.drupal.org/project/%(name)s'
 
-        url = url_template % {'name': project.name}
+        url = url_template % {'name': name}
         regex = REGEX % {'name': project.name}
```
Comment (ralphbean) [inline anitya/lib/backends/drupal6.py]: How about this line?  Should it be just ``name`` as well?

File: anitya/lib/backends/drupal7.py [hunk 2]
```diff
@@ -59,6 +62,10 @@ def get_versions(cls, project):
             when the versions cannot be retrieved correctly
 
         '''
+        name = project.name
+        if 'drupal7:' in name.lower():
+            name = name[len('drupal7:'):].strip()
+
         url_template = 'https://www.drupal.org/project/%(name)s'
 
         url = url_template % {'name': project.name}
```
Comment (ralphbean) [inline anitya/lib/backends/drupal7.py]: And here?

File: anitya/lib/backends/drupal6.py (hunk 1 above)
Comment (pypingou) [inline anitya/lib/backends/drupal6.py]: Yes indeed ² :)

**Comment by ralphbean** [comment]
:+1:

**Comment by pypingou** [comment]
Thanks @ralphbean

**Pull request merged**